OasisDEX/oasis-borrow
# Gf/notificaitions icon

# [Notifications Icon](https://app.shortcut.com/oazo-apps/story/5096/notification-center-ui-component-create-notification-centre-icon)

  
## Changes 👷‍♀️
- Updated header item spacing to match new design
- Added a new notification icon button component that displays a count and triggers the opening and closing of notifications
  
## How to test 🧪
Turn on feature toggle (notifications) and refresh the page. Make sure your wallet is connected and you will see the icon in the top right.
